designer/server/src/main/scala/pl/touk/nussknacker/ui/db/timeseries/StatisticsDb.scala

```diff
@@ -0,0 +1,24 @@
+package pl.touk.nussknacker.ui.db.timeseries
+
+import cats.effect.{IO, Resource}
+
+import scala.concurrent.Future
+import scala.language.higherKinds
+
+trait StatisticsDb[F[_]] extends AutoCloseable {
+  def initialize(): Unit
+  def write(statistics: Map[String, Long]): F[Unit]
+  def read(): F[Map[String, String]]
+}
+
+object StatisticsDb {
+
+  def create(): Resource[IO, StatisticsDb[Future]] =
```

Comment:
because it's related to the specific implementation (QuestDB), shouldn't we move it to `QuestDb` class?